save-cloud-charts/save-cloud/values-minikube.yaml

```diff
@@ -48,3 +48,50 @@ s3:
   endpoint: http://s3
 
 # dependencies
+
+storage:
+  storageClassName: csi-hostpath-sc
+  size: 24Gi
+  annotations:
+    everest.io/disk-volume-type: SAS
+    volume.beta.kubernetes.io/storage-provisioner: hostpath.csi.k8s.io
+
+reposStorage:
+  storageClassName: csi-hostpath-sc
+  annotations:
+    everest.io/disk-volume-type: SAS
+    volume.beta.kubernetes.io/storage-provisioner: hostpath.csi.k8s.io
+
+neo4j:
+  enabled: true
+  neo4j:
+    password: DiktatTestNeo4j@123
```

Comment:
should we commit password?

Reply:
may be apply it in `save-deployment` repo?

Reply:
it's a test one. it's for local deployment only